utils/page_factory.js

```diff
@@ -0,0 +1,24 @@
+const { BasePage } = require("./page_objects/base_page/base_page");
+const { CookiePopup } = require("./page_objects/base_page/cookie_popup");
+const { Homepage } = require("./page_objects/home_page/home_page");
+const { LogIn } = require("./page_objects/login_page/login_page");
+const { SignUp } = require("./page_objects/signup_page/signup_page");
+
+class PageFactory extends BasePage {
```

Comment:
PageFactory DO NOT extends BasePage interface as these classes do different things.